### DISCUSSION TITLE
Deploying btp-resource-consumption-monitor to SAP Kyma Environment

### DISCUSSION THREAD

**Discussion by afzaalmd** [SAP-samples/btp-resource-consumption-monitor#27] [General]
Hello SAP Community,

I was trying to deploy this application into SAP Kyma environment, I have generated the helm chart and deployed using ArgoCD, deployment works fine but some how the application Job Scheduler doesn't creating any Job to fetch the data from Usage Analytics Service. I tries by removing the authorization scopes from cap service to test it. I am getting error code 500.
![image](https://github.com/user-attachments/assets/9ffa93bf-8584-4744-b807-402cf723764d)

Below is the helm chart code: 

`global:
  domain: cluster domain
  imagePullSecret:
    name: token
  image:
    registry: registry.xyz.com/sap-btp/btp-resource-consumption-monitor
    tag: v2.0.56
srv:
  bindings:
    destination:
      serviceInstanceName: 'destination'
    db:
      serviceInstanceName: hana
    auth:
      serviceInstanceName: xsuaa
    btprc-uas:
      serviceInstanceName: btprc-uas
    btprc-notif:
      serviceInstanceName: btprc-notif
    btprc-scheduler:
      serviceInstanceName: btprc-scheduler
  image:
    repository: btp-resource-consumption-srv
  resources:
    limits:
      ephemeral-storage: 1G
      memory: 500M
    requests:
      ephemeral-storage: 1G
      cpu: 500m
      memory: 500M
  health:
    liveness:
      path: /health
    readiness:
      path: /health
hana-deployer:
  image:
    repository: btp-resource-consumption-hana-deployer
  bindings:
    hana:
      serviceInstanceName: hana
  resources:
    limits:
      cpu: 2000m
      memory: 1G
    requests:
      cpu: 1000m
      memory: 1G
html5-apps-deployer:
  env:
    SAP_CLOUD_SERVICE: sap.btp.resourceconsumption
  image:
    repository: btp-resource-consumption-html5-deployer
  bindings:
    xsuaa:
      serviceInstanceName: xsuaa
    destination:
      serviceInstanceName: destination
    html5-apps-repo:
      serviceInstanceName: html5-apps-repo-host
  resources:
    limits:
      cpu: 2000m
      memory: 1G
    requests:
      cpu: 1000m
      memory: 1G
  envFrom:
  - configMapRef:
      name: "{{ .Release.Name }}-html5-apps-deployer-configmap"
backendDestinations:
  btprc-srv-api:
    service: srv
hana:
  serviceOfferingName: hana
  servicePlanName: hdi-shared
html5-apps-repo-host:
  serviceOfferingName: html5-apps-repo
  servicePlanName: app-host
destination:
  serviceOfferingName: 'destination'
  servicePlanName: 'lite'
  parameters:
    version: 1.0.0
    HTML5Runtime_enabled: true
    HTML5.DynamicDestination: true
    init_data:
      subaccount:
        destinations:
        - Name: btprc-srv
          Authentication: OAuth2UserTokenExchange
          TokenServiceInstanceName: btprc-auth
          TokenServiceKeyName: btprc-auth-key
          URL: ~{srv-api/srv-url}
          sap.cloud.service: sap.btp.resourceconsumption
          HTML5.DynamicDestination: true
          WebIDEEnabled: true
          WebIDEUsage: odata_gen
        existing_destinations_policy: update
      instance:
        destinations:
        - Authentication: NoAuthentication
          Name: ui5
          ProxyType: Internet
          Type: HTTP
          URL: https://ui5.sap.com
        - Name: btprc-repo-host
          ServiceInstanceName: btprc-repo-host
          ServiceKeyName: btprc-repo-host-key
          sap.cloud.service: sap.btp.resourceconsumption
        - Name: btprc-auth
          Authentication: OAuth2UserTokenExchange
          ServiceInstanceName: btprc-auth
          ServiceKeyName: btprc-auth-key
          sap.cloud.service: sap.btp.resourceconsumption
        - Name: btprc-notif
          Authentication: OAuth2ClientCredentials
          ServiceInstanceName: btprc-notif
          ServiceKeyName: btprc-notif-key
        - Name: btprc-scheduler
          Authentication: OAuth2ClientCredentials
          ServiceInstanceName: btprc-scheduler
          ServiceKeyName: btprc-scheduler-key
        existing_destinations_policy: update
xsuaa:
  serviceOfferingName: xsuaa
  servicePlanName: application
  jsonParameters: >
    {
      "xsappname": "btp-resource-consumption",
      "tenant-mode": "dedicated"
    }
btprc-uas:
  serviceOfferingName: uas
  servicePlanName: reporting-ga-admin
btprc-notif:
  serviceOfferingName: alert-notification
  servicePlanName: standard
btprc-scheduler:
  serviceOfferingName: jobscheduler
  servicePlanName: standard
  parameters:
    enable-xsuaa-support: true
`

**Comment by willempardaens** [comment]
Created PR #30 for Kyma deployments.
